package.json

```diff
@@ -25,6 +25,7 @@
     "gulp-uglify": "^1.5.4",
     "gulp-util": "^3.0.7",
     "merge-stream": "^1.0.0",
+    "prop-types": "^15.5.8",
```

Comment:
this should be added as a dependency, otherwise projects depending on `react-hammerjs` won't have it installed